packages/eslint-config-sentry-app/package.json

```diff
@@ -29,7 +29,7 @@
     "@typescript-eslint/parser": "^5.17.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-config-sentry": "^1.93.0",
-    "eslint-config-sentry-react": "^1.93.0",
+    "eslint-config-sentry-react": "^1.94.0",
```

Comment:
@JonasBa it looks like this should be `^1.93.0`, as `1.94.0` doesn't actually exist: https://www.npmjs.com/package/eslint-config-sentry-react

Also see the version changes in `yarn.lock` files :D